src/Metadata/Driver/AnnotationOrAttributeDriver.php

```diff
@@ -309,7 +311,12 @@ protected function getClassAnnotations(\ReflectionClass $class): array
                 static function (\ReflectionAttribute $attribute): object {
                     return $attribute->newInstance();
                 },
-                $class->getAttributes()
+                array_filter(
+                    $class->getAttributes(),
+                    static function (\ReflectionAttribute $attribute): bool {
+                        return class_exists($attribute->getName());
+                    }
+                )
```

Comment:
The same filtering is needed in this class' `getMethodAnnotations()` and `getPropertyAnnotations()` methods.

Reply:
Ah good catch, I've seen that in the `AttributeDriver` but havent adapted in this one.